### PR TITLE
WF-65875 Extract accessibility tags and get image from custom rubber stamp

### PR DESCRIPTION
**Pull request by AnanthaGokulaRaman**
Hi @chinnumuniyappan,

Kindly review the UG documentation for Extract accessibility tags and get image from custom rubber stamp.
Content review: https://syncfusion.atlassian.net/browse/DOC-5261

Regards,
Anantha Gokula Raman J

### CODE REVIEW

File: File-Formats/PDF/Working-with-Tagged-PDF.md [hunk 1]
```diff
@@ -2906,3 +2906,184 @@ Follow the below steps to validate the tagged PDF information in Adobe Acrobat:
 The following screenshot shows the Accessibility checker dialog box.
 
 ![TaggedPDF](TaggedPDF_images/TaggedPDF_img0.png) 
+
+## Get the accessibility tags
+
+Added support for extracting accessibility tags from PDF documents.
```
Comment (chinnumuniyappan) [inline File-Formats/PDF/Working-with-Tagged-PDF.md:2912]: You can extract the existing tag details by using the **StructureElement** property available on the **PdfLoadedDocument** class. The following code explains this.

**Review by chinnumuniyappan — CHANGES_REQUESTED** [review]

File: File-Formats/PDF/Working-with-Tagged-PDF.md (hunk 1 above)
Comment (AnanthaGokulaRaman) [reply File-Formats/PDF/Working-with-Tagged-PDF.md]: Hi @chinnumuniyappan,

The suggested changes has been updated.

Regards,
Anantha Gokula Raman J

File: File-Formats/PDF/Working-with-Tagged-PDF.md [hunk 2]
```diff
@@ -2906,3 +2906,184 @@ Follow the below steps to validate the tagged PDF information in Adobe Acrobat:
...
+{% highlight c# tabtitle="C#" %}
+
+//Load the existing PDF document.
+PdfLoadedDocument document = new PdfLoadedDocument("TaggedPDF.pdf");
+//Get the structure element root from the document.
+PdfStructureElement rootElement = document.StructureElement;
+//Get the first page from the document.
+PdfLoadedPage loadedPage = document.Pages[0] as PdfLoadedPage;
+//Get the structure elements associated with the page.
+PdfStructureElement[] pageElements = loadedPage.StructureElements;
```
Comment (chinnumuniyappan) [inline File-Formats/PDF/Working-with-Tagged-PDF.md:2925]: This should be split as a separate topic.

You can also extract the accessibility tags page-wise with the help of the StructureElement property available on the PdfLoadedPage class. The following code illustrates this.

**Review by chinnumuniyappan — CHANGES_REQUESTED** [review]

File: File-Formats/PDF/Working-with-Tagged-PDF.md (hunk 2 above)
Comment (AnanthaGokulaRaman) [reply File-Formats/PDF/Working-with-Tagged-PDF.md:3073]: Hi @chinnumuniyappan,

The suggested changes has been updated.

Regards,
Anantha Gokula Raman J

**Review by chinnumuniyappan — APPROVED** [review]

**Pull request merged**